backend/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "dlp-backend"
+version = "0.1.0"
+description = "Main Backend for Deep Learning Playground's Training Services"
+authors = ["DSGT-DLP <dlp@datasciencegt.org>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "dlp_backend"}]
+
+# Try not directly edit this file! Use `poetry add` and `poetry remove` instead.
+
+[tool.poetry.dependencies]
+python = ">=3.9, <3.10"
+flask-socketio = "*"
+numpy = "*"
+pandas = "*"
+matplotlib = "*"
+seaborn = "*"
+mypy = "*"
+pytest = "*"
+scikit-learn = "*"
+scipy = "1.7.3"
+pyyaml = "*"
+statsmodels = "*"
+torch = "*"
+transformers = "*"
+pre-commit = "*"
+torchvision = "*"
+timm = "*"
+wwf = "*"
+fastai = "==2.7.4"
+firebase_admin = "*"
+pyrebase4 = "*"
+moto = "*"
+docker = "*"
```

Comment:
do we actually have a use case for `pip install docker` in our codebase. I wonder if this dependency is actually needed